### PR TITLE
Users can pay with Stripe

### PR DESCRIPTION
**Pull request by jyuw**
#### Description
Changes proposed in this pull request:
* adds subscription page
* adds Stripe functionality
 
#### What I have learned working on this feature:
* using the Stripe gem
* using Chromedriver and Selenium
* Rails Credentials

#### Screenshots:
screen shot 2018-03-21 at 12 14 34 pm

### CODE REVIEW

**Review by kmthorsnes — DISMISSED** [review]
Looks good to me

**Review by davidboll — DISMISSED** [review]
Noting to complain about, good job!

File: config/routes.rb [hunk 1]
```diff
@@ -5,4 +5,6 @@
   end
   resources :categories, only: [:show]
   devise_for :users
+  resources :subscriptions, only: [:new]
+  post '/charges', controller: :subscriptions, action: :create
```
Comment (magnus-thor) [inline config/routes.rb]: try to use restful routes, add `:create` to resources and update the url in the view.

**Review by kmthorsnes — APPROVED** [review]
Now it looks even better!

**Review by magnus-thor — APPROVED** [review]
Good job

**Pull request merged**